Backend/app/rider-platform/rider-app/Main/src/Domain/Action/Dashboard/Customer.hs

```diff
@@ -92,9 +94,9 @@ unblockCustomer merchantShortId customerId = do
   -- merchant access checking
   let merchantId = customer.merchantId
   unless (merchant.id == merchantId) $ throwError (PersonDoesNotExist personId.getId)
-
+  Redis.del (mkCancellationKey customerId.getId)
```

Comment:
hey, this will not work directly. You will have to create a new function in shared-kernel. See the implementation of getCurrentWindowValues, create a similar fn called deleteCurrentWindowValues and use that here.